app/client/components/creditTransfer/creditTransferList.jsx

```diff
@@ -124,6 +108,29 @@ class CreditTransferList extends React.Component {
     }
   }
 
+  // handles credit transfer ids array
+  sortCreditTransfers = (creditTransferIds) => {
+
+    creditTransferIds.map(i => {
+
+      // retrieve any sender account that has not been retrieved
+      let sender_account_id = this.props.creditTransfers.byId[i].sender_transfer_account_id
+      if(!this.props.transferAccounts.byId[sender_account_id]){
+        if(this.props.creditTransfers.byId[i].transfer_type !== "DISBURSEMENT"){
+          this.props.loadTransferAccountList(sender_account_id)
+        }
+      }
```

Comment:
upon component load, this will check if any sender accounts have not been retrieved. I excluded disbursement credit transfers because it loads a blank vendor account